.github/workflows/ci-test.yml

```diff
@@ -15,6 +16,10 @@ jobs:
       with:
         submodules: true
```

Comment:
@yurishkuro - there are no submodules in this repo, right? If so, then the two occurrences of `with: submodules: true` can be dropped.

Reply:
See: 

- #905